.github/workflows/tests.yaml

```diff
@@ -261,8 +258,38 @@ jobs:
           # build, so simply inject an explicit no-op test target.
           echo "//scripts:no_op_test" >> $TARGETS_FILE
 
-          bazelisk test -c ${{ matrix.build_mode }} \
-            --target_pattern_file=$TARGETS_FILE
+          for i in {1..10}; do
```

Comment:
10 seems like a lot? It seems like we should be made aware if this many retries are needed, and make adjustments like reducing `--jobs`. Maybe 4 would make more sense?

Reply:
I'd much rather the actions finally succeed, especially with the merge queue... And we should still be able to find these because if it's a problem the action should show up as significantly slow?

Still, dropped this to 5, with 3 at the high jobs count (to try to get as much done with high throughput as we can), and then 2 more with a low jobs count as a last-ditch approach.